Makefile

```
@@ -77,8 +77,9 @@ devnet-up:
 	PYTHONPATH=./bedrock-devnet python3 ./bedrock-devnet/main.py --monorepo-dir=.
 .PHONY: devnet-up
 
+# alias for devnet-up
 devnet-up-deploy:
-	PYTHONPATH=./bedrock-devnet python3 ./bedrock-devnet/main.py --monorepo-dir=. --deploy
+	PYTHONPATH=./bedrock-devnet python3 ./bedrock-devnet/main.py --monorepo-dir=.
 .PHONY: devnet-up-deploy
```

If `devnet-up-deploy` is now the same as `devnet-up`, can we just remove it or if not point `devnet-up-deploy` to `devnet-up`.

ie.
```suggestion
devnet-up-deploy: devnet-up
```

---

+1 to removing `devnet-up-deploy`

---

this is now fixed, good call